omod/src/main/java/org/openmrs/web/dwr/DWRConceptService.java

```diff
@@ -236,7 +236,11 @@
 	 * @param conceptId the id to look for
 	 * @return a {@link ConceptListItem} or null if conceptId is not found
 	 */
-	public ConceptListItem getConcept(Integer conceptId) {
```

Comment:
What is the reason behind changing from the integer conceptId to string phrase as the method parameter?

Reply:
the issue is that,  the cause of the error was arising from the fact that the value to be taken as a conceptId here ` getConcept(Integer conceptId)` was in the form say "Concept #33" for id 33, "Concept #6" for id 6,, hence  say "Concept #66" couldnt be converted to a conceptid ,then hats why it was throwing the javascript error described in the ticket. 
So i changed from integer to string , so that the value  eg "Concept #66" be recieved as a string , then i trimm off the "Concept #" to remain with a Numeric string "66" , and then i convert it back to a concept id